package.json

```diff
@@ -30,6 +30,9 @@
         "node-polyfill-webpack-plugin": "^1.1.4",
         "qs": "^6.5.2",
         "quasar": "^2.0.4",
+        "quasar-app-extension-f-paginate": "file:../freedom-components/f-paginate",
```

Comment:
Shouldn't we be getting this from the repo?

Reply:
Will update how we include on package json once the freedom components are tested and merged